Enterprise-CMCS/macpro-mdct-mfp
# Demo separate modals per report type

### Description

Compare to PR #605 


### Related ticket(s)


CMDCT-3473 

---
### How to test

Deployed env: https://d2hu46llnelba1.cloudfront.net/
Fastest to test locally with the postman collection:
- Log in as a state user
- Create a WP and ensure you get the first work plan form (asks for year and period)
- Use the postman collection to fill out, submit, and approve the report
- In the UI, create a copy WP and ensure the correct modal shows
- Go to SAR dashboard and ensure that modal works as expected


### Notes




---
### Pre-review checklist
